Failed to load ets-wms-client13 ctl form

Tested with teamengine current master and ets-wms-client13 and got the following error:

```
Test Name : main:wms-client type Mandatory
net.sf.saxon.s9api.SaxonApiException: Error in call to extension function {public org.w3c.dom.Node com.occamlab.te.TECore.form(org.w3c.dom.Document,java.lang.String) throws java.lang.Exception}: Exception in extension function java.net.MalformedURLException: no protocol: images/minus.png
	at net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:192)
	at com.occamlab.te.TECore.executeTemplate(TECore.java:613)
	at com.occamlab.te.TECore.executeTest(TECore.java:824)
	at com.occamlab.te.TECore.execute_test(TECore.java:432)
	at com.occamlab.te.TECore.execute_suite(TECore.java:480)
	at com.occamlab.te.TECore.execute(TECore.java:309)
	at com.occamlab.te.TECore.run(TECore.java:2417)
	at java.lang.Thread.run(Thread.java:748)
Caused by: net.sf.saxon.trans.XPathException: Error in call to extension function {public org.w3c.dom.Node com.occamlab.te.TECore.form(org.w3c.dom.Document,java.lang.String) throws java.lang.Exception}: Exception in extension function java.net.MalformedURLException: no protocol: images/minus.png
	at net.sf.saxon.functions.ExtensionFunctionCall.iterate(ExtensionFunctionCall.java:151)
	at net.sf.saxon.instruct.CopyOf.processLeavingTail(CopyOf.java:292)
	at net.sf.saxon.instruct.Choose.processLeavingTail(Choose.java:625)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.expr.LetExpression.processLeavingTail(LetExpression.java:551)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.expr.LetExpression.processLeavingTail(LetExpression.java:551)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.expr.LetExpression.processLeavingTail(LetExpression.java:551)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Instruction.process(Instruction.java:94)
	at net.sf.saxon.instruct.ForEach.processLeavingTail(ForEach.java:300)
	at net.sf.saxon.expr.LetExpression.processLeavingTail(LetExpression.java:551)
	at net.sf.saxon.instruct.Block.processLeavingTail(Block.java:409)
	at net.sf.saxon.instruct.Template.applyLeavingTail(Template.java:175)
	at net.sf.saxon.instruct.ApplyTemplates.applyTemplates(ApplyTemplates.java:343)
	at net.sf.saxon.Controller.transformDocument(Controller.java:1736)
	at net.sf.saxon.Controller.transform(Controller.java:1560)
	at net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:190)
	... 7 more
Caused by: java.net.MalformedURLException: no protocol: images/minus.png
	at java.net.URL.<init>(URL.java:593)
	at java.net.URL.<init>(URL.java:490)
	at java.net.URL.<init>(URL.java:439)
	at com.occamlab.te.form.ImageHandler.saveImages(ImageHandler.java:56)
	at com.occamlab.te.TECore.form(TECore.java:2319)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.sf.saxon.functions.ExtensionFunctionCall.invokeMethod(ExtensionFunctionCall.java:533)
	at net.sf.saxon.functions.ExtensionFunctionCall.call(ExtensionFunctionCall.java:256)
	at net.sf.saxon.functions.ExtensionFunctionCall.iterate(ExtensionFunctionCall.java:147)
	... 33 more
```

--- a/teamengine-core/src/main/java/com/occamlab/te/form/ImageHandler.java
+++ b/teamengine-core/src/main/java/com/occamlab/te/form/ImageHandler.java
@@ -50,6 +50,7 @@ public class ImageHandler {
         for ( Element image : images ) {
             if ( image.hasAttribute( "src" ) ) {
                 String src = image.getAttribute( "src" );
+                if(src.contains("http")){
                 String imageFormat = parseImageFormat( src );
                 String testName = System.getProperty( "TestName" );
                 String imgName = parseImageName( testName );
@@ -61,6 +62,7 @@ public class ImageHandler {
                 }
                 ImageIO.write( img, imageFormat, file );
                 imageCount++;
+                }
             }
         }
     }

--- a/teamengine-core/src/main/java/com/occamlab/te/form/ImageHandler.java
+++ b/teamengine-core/src/main/java/com/occamlab/te/form/ImageHandler.java
@@ -8,6 +8,7 @@ import java.net.MalformedURLException;
 import java.net.URL;
 import java.net.URLDecoder;
 import java.util.List;
+import java.util.logging.Logger;
 
 import javax.imageio.ImageIO;
 
@@ -19,6 +20,8 @@ import com.occamlab.te.util.DomUtils;
  * @author <a href="mailto:goltz@lat-lon.de">Lyn Goltz </a>
  */
 public class ImageHandler {
+
+    private static final Logger LOG = Logger.getLogger( ImageHandler.class.getName() );
 
     private final File testLogDir;
 
@@ -50,27 +53,35 @@ public class ImageHandler {
         for ( Element image : images ) {
             if ( image.hasAttribute( "src" ) ) {
                 String src = image.getAttribute( "src" );
-                if(src.contains("http")){
-                String imageFormat = parseImageFormat( src );
-                String testName = System.getProperty( "TestName" );
-                String imgName = parseImageName( testName );
-                URL url = new URL( src );
-                BufferedImage img = ImageIO.read( url );
-                File file = createImageFile( imageCount, imageFormat, imgName );
-                if ( !file.exists() ) {
-                    file.getParentFile().mkdirs();
-                }
-                ImageIO.write( img, imageFormat, file );
-                imageCount++;
+                if ( src.startsWith( "http" ) ) {
+                    String imageFormat = parseImageFormat( src );
+                    String imgName = parseImageName();
+                    File target = createImageFile( imageCount, imageFormat, imgName );
+                    saveImage(src, imageFormat, target);
+                    imageCount++;
                 }
             }
+        }
+    }
+
+    private void saveImage( String src, String imageFormat, File file ) {
+        try {
+            URL url = new URL( src );
+            BufferedImage img = ImageIO.read( url );
+            ImageIO.write( img, imageFormat, file );
+        } catch ( Exception e ) {
+            LOG.warning( "Could not write image " + src + " to " + file );
         }
     }
 
     private File createImageFile( int imageCount, String imageFormat, String imgName ) {
         String downloadPath = testLogDir + File.separator + sessionId + File.separator + "images" + File.separator
                               + imgName + imageCount + "." + imageFormat;
-        return new File( downloadPath );
+        File target = new File( downloadPath );
+        if ( !target.exists() ) {
+            target.getParentFile().mkdirs();
+        }
+        return target;
     }
 
     private String parseImageFormat( String src )
@@ -87,7 +98,8 @@ public class ImageHandler {
         return "PNG";
     }
 
-    private String parseImageName( String testName ) {
+    private String parseImageName() {
+        String testName = System.getProperty( "TestName" );
         if ( testName.contains( " " ) ) {
             return testName.substring( testName.indexOf( ":" ) + 1, testName.indexOf( " " ) );
         }

